tests/_integration/table_template.txt

```diff
@@ -0,0 +1,6 @@
+#       | python | go | ruby | js | php
```

Comment:
is this really needed?

Reply:
it's helped me to distribute such checklist across other scripts
but if you think it's unnecessary I will drop it